src/meeting/meet/speakersObserver.ts

```
@@ -0,0 +1,675 @@
+
+                // setupMutationObserver
+                async function setupMutationObserver(): Promise<boolean> {
+                    try {
+                        const observe_parameters = await getSpeakerRootToObserve(recordingMode)
+
+                        if (!observe_parameters || !observe_parameters[0]) {
+                            console.warn('[Meet-Browser] No valid root element to observe')
+                            return false
+                        }
+
+                        MUTATION_OBSERVER!.disconnect()
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)